linera-execution/src/lib.rs

```diff
@@ -13,6 +13,8 @@ mod policy;
 mod resources;
 mod runtime;
 pub mod system;
+#[cfg(any(test, feature = "test"))]
```

Comment:
Optional: we might want to create a `with_testing` alias in `linera-execution/build.rs` to be consistent with the changes that are being done in the other crates (e.g., `linera-witty`).